SharpGrad/Value.cs

```diff
@@ -5,7 +5,7 @@ public class Value
         private readonly double _value;
         private List<Value> _children = new List<Value>();
         private Action _backward = () => { };
-        private double _grad;
+        private double _grad = 0;
```

Comment:
<!-- Comment from Crocodile -->
Comment from [Crocodile 🐊](https://app.crocodile.dev/reviews/rwPWqdvJHMJe?change=SharpGrad%2FValue.cs&iter=1#thread-thQFAN3nXfnr) on unchanged code outside of the pull request:

https://github.com/jlao/SharpGrad/blob/a6b2fd158657e26e99b29ea03d67040387729c43/SharpGrad/Value.cs#L15

how your new changes are going to work with this line?